proto/sedachain/tally/v1/tally.proto

```diff
@@ -34,4 +34,6 @@ message Params {
   ];
   // MaxResultSize is the maximum size of the result of a data request in bytes.
   uint32 max_result_size = 8;
+  // MaxRevealDataLength is the maximum length of a reveal data.
+  uint32 max_reveal_data_length = 9;
```

Comment:
I don't think we need this parameter. For now at least we probably only want to limit the size of the node we're extracting in the std-dev filter to prevent expensive big number computation. We can leave this as a hardcoded variable, since changing it will most likely require other changes to prevent abuse. What do you think?

Reply:
I'm down to get rid of reveal data length check altogether since we already have some limit on the reveal size in the contract. And the std dev filter limits the big int size at 256 too.